packages/typespec-client-generator-core/src/types.ts

```diff
@@ -437,9 +437,12 @@ export function getSdkModel(context: SdkContext, type: Model, operation?: Operat
   const httpOperation = operation
     ? ignoreDiagnostics(getHttpOperation(context.program, operation))
     : undefined;
-  const isFormDataType = httpOperation
-    ? Boolean(httpOperation.parameters.body?.contentTypes.includes("multipart/form-data"))
-    : false;
+  const httpBody = httpOperation?.parameters.body;
+  let isFormDataType = false;
+  if (httpBody) {
+    const isMultipartOperation = httpBody.contentTypes.includes("multipart/form-data");
```

Comment:
shouldn't you just check for startwith`multipart/` there is `multtipart/mixed` too

Reply:
ooh good point